pkg/jobrunaggregator/jobrunaggregatorlib/ci_data_client.go

```diff
@@ -92,39 +92,47 @@ func NewCIDataClient(dataCoordinates BigQueryDataCoordinates, client *bigquery.C
+	// which updates the static files in origin.
 	queryString := c.dataCoordinates.SubstituteDataSetLocation(`
-    SELECT
-    BackendName,
-    Release,
-    FromRelease,
-    Platform,
-    Architecture,
-    Network,
-    Topology,
-    IFNULL(SAFE_CAST(ANY_VALUE(P95) AS STRING), "0.0") AS P95,
-    IFNULL(SAFE_CAST(ANY_VALUE(P99) AS STRING), "0.0") AS P99,
-    FROM (
-        SELECT
-            Jobs.Release,
-            Jobs.FromRelease,
-            Jobs.Platform,
-            Jobs.Architecture,
-            Jobs.Network,
-            Jobs.Topology,
-            BackendName,
-            PERCENTILE_CONT(BackendDisruption.DisruptionSeconds, 0.95) OVER(PARTITION BY BackendDisruption.BackendName, Jobs.Network, Jobs.Platform, Jobs.Release, Jobs.FromRelease, Jobs.Topology) AS P95,
-            PERCENTILE_CONT(BackendDisruption.DisruptionSeconds, 0.99) OVER(PARTITION BY BackendDisruption.BackendName, Jobs.Network, Jobs.Platform, Jobs.Release, Jobs.FromRelease, Jobs.Topology) AS P99,
-        FROM
-            DATA_SET_LOCATION.BackendDisruption as BackendDisruption
-        INNER JOIN
-            DATA_SET_LOCATION.BackendDisruption_JobRuns as JobRuns on JobRuns.Name = BackendDisruption.JobRunName
-        INNER JOIN
-            DATA_SET_LOCATION.Jobs as Jobs on Jobs.JobName = JobRuns.JobName
-        WHERE
-            JobRuns.StartTime > TIMESTAMP_SUB(CURRENT_TIMESTAMP(), INTERVAL 21 DAY)
-    )
-    GROUP BY
-        BackendName, Release, FromRelease, Platform, Architecture, Network, Topology
+	SELECT * FROM (
+		SELECT
+		COUNT(*) AS JobRuns,
+		BackendName,
+		Release,
+		FromRelease,
+		Platform,
+		Architecture,
+		Network,
+		Topology,
+		IFNULL(SAFE_CAST(ANY_VALUE(P95) AS STRING), "0.0") AS P95,
+		IFNULL(SAFE_CAST(ANY_VALUE(P99) AS STRING), "0.0") AS P99
+		FROM (
+			SELECT
+				Jobs.Release,
+				Jobs.FromRelease,
+				Jobs.Platform,
+				Jobs.Architecture,
+				Jobs.Network,
+				Jobs.Topology,
+				BackendName,
+				PERCENTILE_CONT(BackendDisruption.DisruptionSeconds, 0.95) OVER(PARTITION BY BackendDisruption.BackendName, Jobs.Network, Jobs.Platform, Jobs.Release, Jobs.FromRelease, Jobs.Topology) AS P95,
+				PERCENTILE_CONT(BackendDisruption.DisruptionSeconds, 0.99) OVER(PARTITION BY BackendDisruption.BackendName, Jobs.Network, Jobs.Platform, Jobs.Release, Jobs.FromRelease, Jobs.Topology) AS P99,
+			FROM
+				DATA_SET_LOCATION.BackendDisruption as BackendDisruption
+			INNER JOIN
+				DATA_SET_LOCATION.BackendDisruption_JobRuns as JobRuns on JobRuns.Name = BackendDisruption.JobRunName
+			INNER JOIN
+				DATA_SET_LOCATION.Jobs as Jobs on Jobs.JobName = JobRuns.JobName
+			WHERE
+				JobRuns.StartTime > TIMESTAMP_SUB(CURRENT_TIMESTAMP(), INTERVAL 21 DAY)
+		)
+		GROUP BY
+			BackendName, Release, FromRelease, Platform, Architecture, Network, Topology
+	)
+	WHERE JobRuns >= 100
+	ORDER BY Release, FromRelease, Platform, Architecture, Network, Topology, BackendName
```

Comment:
Just a note on this one, we currently order the jobs based off a generated key using these values when we are creating the resulting json file, so this might not be needed. It's this  [sort](https://github.com/openshift/ci-tools/blob/8999351f7ea53c2fe6099f8211d32550ad821590/pkg/jobrunaggregator/jobrunhistoricaldataanalyzer/util.go#L166-L168) driven by the key created [here](https://github.com/openshift/ci-tools/blob/8999351f7ea53c2fe6099f8211d32550ad821590/pkg/jobrunaggregator/jobrunaggregatorapi/types_row_historical_data.go#L45-L55). So far from the recent PR that should keep things consistent more or less (depending on how many jobs get removed or added in a given PR). If you expand the diff in this [PR](https://github.com/openshift/origin/pull/27475/files) does it represent what you were looking to achieve @dgoodwin  ?

Reply:
yes that's great, thanks Egli, forgot we arranged that already.